lighthouse-core/lib/large-libraries.json

```diff
@@ -0,0 +1 @@
ment","scoped":false,"size":237265,"version":"2.24.0"},"2.23.0":{"dependencyCount":0,"dependencySizes":[{"approximateSize":329990,"name":"moment"},{"approximateSize":352,"name":"webpack"}],"description":"Parse, validate, manipulate, and display dates","gzip":66675,"hasJSModule":false,"hasJSNext":"./src/moment.js","hasSideEffects":true,"name":"moment","repository":"https://github.com/moment/moment","scoped":false,"size":261992,"version":"2.23.0"}},"dayjs":{"1.8.29":{"assets":[{"gzip":2849,"name":"main","size":6690,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":6578,"name":"dayjs"}],"description":"2KB immutable date time library alternative to Moment.js with the same modern API ","gzip":2849,"hasJSModule":"dayjs.min.js","hasJSNext":false,"hasSideEffects":true,"name":"dayjs","repository":"https://github.com/iamkun/dayjs.git","scoped":false,"size":6690,"version":"1.8.29"},"latest":{"assets":[{"gzip":2849,"name":"main","size":6690,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":6578,"name":"dayjs"}],"description":"2KB immutable date time library alternative to Moment.js with the same modern API ","gzip":2849,"hasJSModule":"dayjs.min.js","hasJSNext":false,"hasSideEffects":true,"name":"dayjs","repository":"https://github.com/iamkun/dayjs.git","scoped":false,"size":6690,"version":"1.8.29"}},"luxon":{"1.24.1":{"assets":[{"gzip":20912,"name":"main","size":70947,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":107210,"name":"luxon"}],"description":"Immutable date wrapper","gzip":20912,"hasJSModule":"src/luxon.js","hasJSNext":false,"hasSideEffects":true,"name":"luxon","repository":"https://github.com/moment/luxon.git","scoped":false,"size":70947,"version":"1.24.1"},"latest":{"assets":[{"gzip":20912,"name":"main","size":70947,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":107210,"name":"luxon"}],"description":"Immutable date wrapper","gzip":20912,"hasJSModule":"src/luxon.js","hasJSNext":false,"hasSideEffects":true,"name":"luxon","repository":"https://github.com/moment/luxon.git","scoped":false,"size":70947,"version":"1.24.1"}},"date-fns":{"2.14.0":{"assets":[{"gzip":18304,"name":"main","size":84293,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":178180,"name":"date-fns"}],"description":"Modern JavaScript date utility library","gzip":18304,"hasJSModule":"esm/index.js","hasJSNext":false,"hasSideEffects":false,"name":"date-fns","repository":"https://github.com/date-fns/date-fns.git","scoped":false,"size":84293,"version":"2.14.0"},"latest":{"assets":[{"gzip":18304,"name":"main","size":84293,"type":"js"}],"dependencyCount":0,"dependencySizes":[{"approximateSize":178180,"name":"date-fns"}],"description":"Modern JavaScript date utility library","gzip":18304,"hasJSModule":"esm/index.js","hasJSNext":false,"hasSideEffects":false,"name":"date-fns","repository":"https://github.com/date-fns/date-fns.git","scoped":false,"size":84293,"version":"2.14.0"}}}
```

Comment:
seems we could filter out most of these properties

Reply:
kk

I kept a few like repository/description so we can hyperlink the library name and show a description too